change/@ni-nimble-tokens-5ebfa1b9-f46d-48c9-b259-0f1230a4d540.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Added 'down-right-from-square', 'up-right-from-square', and 'file-arrow-curved-right' icons. Removed 'share-square' (use the 'up-right-from-square' icon instead) and 'file-export' (use the 'file-arrow-curved-right' icon instead) icons.",
+  "packageName": "@ni/nimble-tokens",
+  "email": "1458528+fredvisser@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
Thought to self / @jattasNI maybe this should have been a major dependent change type?
I'm not sure how it cascades but now Blazor and Nimble Angular have a breaking change because components are being removed in both.